### DISCUSSION TITLE
Incomplete answer

### DISCUSSION THREAD

**Discussion by lijunpeng259** [infiniflow/ragflow#1441] [Q&A]
[工作簿1.xlsx](https://github.com/user-attachments/files/16139782/1.xlsx)
1.已向知识库上传“工作簿1.xlsx"文件。
2.提问，如图：
![image](https://github.com/infiniflow/ragflow/assets/175092223/6472cf79-cc28-471e-942f-efde7dc38806)

3.回复内容不完整。未完整的回复知识库里内容。答复出现中断。尝试过多次，基本再第15条就中断。

对于如上情况，请问是否有大神遇到。要如何做调整。

**Comment by KevinHuSh** [comment]
https://github.com/infiniflow/ragflow/issues/1446
